prow/cluster/boskos-metrics.yaml

```diff
@@ -22,8 +22,12 @@ spec:
     app: boskos-metrics
   ports:
   - name: default
-    protocol: TCP
     port: 80
+    protocol: TCP
+    targetPort: 8080
```

Comment:
This is intentional: the boskos-metrics service proxies the main `/metric` API on port 8080, and this was masked in https://github.com/kubernetes/test-infra/pull/15356. Configuring the scraper to look at port 9090 (also included here) is a better fix.

Reply:
It used to be exposed, prior to #15356. it basically just proxies the `/metric` API:

https://github.com/kubernetes/test-infra/blob/0a8437ab4b6cf302d8e170b945424d532fdb09e8/boskos/metrics/metrics.go#L131-L170

Nobody has complained that this is missing, so maybe we don't need it.

Reply:
(removed)